### PR TITLE
Feature/updates

### PR DESCRIPTION
**Pull request by dingo-d**
Minor updates to handbook:

Sonarqube chapter updates
PHP standards updates
Versioning updates
Dockerfile updates

### CODE REVIEW

File: Automated testing in WordPress/Unit testing.md [hunk 1]
```diff
@@ -132,9 +132,97 @@ Brain Monkey allows to mock WordPress function (just like any PHP function), and
...
+```
+
 ## Useful links
 
-[Unit Tests for PHP code](https://inpsyde.com/en/php-unit-tests-without-wordpress/)
-[An Introduction To Automated Testing Of WordPress Plugins With PHPUnit](https://www.smashingmagazine.com/2017/12/automated-testing-wordpress-plugins-phpunit/)
-[Unit Tests for WordPress Plugins](https://pippinsplugins.com/unit-tests-wordpress-plugins-introduction/)
-[An introduction to unit testing (for WordPress)](https://tfrommen.de/an-introduction-to-unit-testing-for-wordpress/)
+[Unit Tests for PHP code](https://inpsyde.com/en/php-unit-tests-without-wordpress/)  
+[An Introduction To Automated Testing Of WordPress Plugins With PHPUnit](https://www.smashingmagazine.com/2017/12/automated-testing-wordpress-plugins-phpunit/)
```
Comment (IvanGrginovInf) [inline Automated testing in WordPress/Unit testing.md]: Maybe we could (in the future) add a _how to setup everything_ section / tutorial (basically this tutorial but in the handbook - or just emphasise it).

File: Coding standards/PHP coding standards.md [hunk 2]
```diff
@@ -24,8 +24,26 @@ namespace Inf_Theme;
 
 Every folder which holds classes constitutes a subnamespace.
 
+Be aware that WordPress 'lives' in a global namespace. So, when using functions and classes from the WordPress core, it's necessary to put a slash in front of class or function name, so that it's called from the global namespace. For example, calling `WP_Query` inside your class should be done like sources
+
+```php
+$query = new \WP_Query( $arguments );
```
Comment (IvanGrginovInf) [inline Coding standards/PHP coding standards.md]: Could we maybe add a phpcs rule that throws a warning / error when not doing this?

File: Development environment/Docker.md [hunk 3]
```diff
@@ -170,6 +170,10 @@ Remove all images
 
 `docker rmi $(docker images -q)`
 
+Prune system from unused images and containers (use with caution!)
```
Comment (IvanGrginovInf) [inline Development environment/Docker.md]: `use with caution`. Why? What could happen? Just reading it like this makes me not want to run this command ever. (Tho this is just my initial reaction and I have 0 docker experience so take this with a grain of salt). Maybe a bit more detailed caution explanation.

Comment (dingo-d) [reply Development environment/Docker.md]: I've updated this part. It's tricky as you may remove things you maybe don't want to remove.

Comment (IvanGrginovInf) [reply Development environment/Docker.md]: Much better tho no need to remove the caution. It was just missing some extra info (like the documentation link you added) as to why it's a dangerous command.

File: Automated testing in WordPress/Unit testing.md (hunk 1 above)
Comment (dingo-d) [reply Automated testing in WordPress/Unit testing.md]: General helpers are tricky as they are `static` xD I'll add them in due time :D

**Pull request merged**